def/fi/menu_linux.cson

```diff
@@ -0,0 +1,305 @@
+Menu:
+  "&File":
+    value: "Tiedosto"
```

Comment:
Please preserve hotkey prompt (in this case `&F`) in the word or wrapped in braces

for example, [`value: "Datei(&F)"` (de/menu_linux.cson)](https://github.com/liuderchi/atom-i18n/blob/master/def/de/menu_linux.cson#L3)

Ref. [no3 in CONTRIBUTING.md](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide)